File: SD_TOP/Enaki/Examen Practic/Modele_2020/microservicii-spring-kotlin/SQLiteExample_maven/src/main/kotlin/org/example/services/BeerRowMapper.java
package org.example.services;

import kotlin.jvm.Throws;

import java.sql.SQLException;

class BeerRowMapper{

}
/*
class BeerRowMapper: RowMapper<Beer?>{
    @Throws(SQLException::class)
    override fun mapRow(rs: ResultSet, rowNum: Int): Beer {
        return Beer(rs.getInt("id"), rs.getString("name"), rs.getFloat("price"))
    }
}
 */

File: SD_TOP/Enaki/Examen Practic/Modele_2020/microservicii-spring-kotlin/example-kotlin/src/main/kotlin/org/example/services/BeerRowMapper.kt
package org.example.services

import org.example.model.Beer
import org.springframework.jdbc.core.RowMapper
import java.sql.ResultSet
import java.sql.SQLException
import kotlin.jvm.Throws

class BeerRowMapper: RowMapper<Beer?> {
    @Throws(SQLException::class)
    override fun mapRow(rs: ResultSet, rowNum: Int): Beer {
        return Beer(rs.getInt("id"), rs.getString("name"), rs.getFloat("price"))
    }
}